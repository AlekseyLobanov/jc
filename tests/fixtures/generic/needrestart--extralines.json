{"version":"3.5","running_kernel_version":"5.10.0-19-amd64","expected_kernel_version":"5.10.0-20-amd64","kernel_status":3,"service":["exim4.service","NetworkManager.service","packagekit.service"]}
