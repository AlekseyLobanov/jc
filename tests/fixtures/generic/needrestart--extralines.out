NEEDRESTART-VER: 3.5
NEEDRESTART-KCUR: 5.10.0-19-amd64
NEEDRESTART-KEXP: 5.10.0-20-amd64
NEEDRESTART-KSTA: 3
NEEDRESTART-UCSTA: 1
NEEDRESTART-UCCUR: 0x08108109
Use of uninitialized value $ucode_vars{"AVAIL"} in concatenation (.) or string at /usr/sbin/needrestart line 904.
NEEDRESTART-UCEXP: 
NEEDRESTART-SVC: exim4.service
NEEDRESTART-SVC: NetworkManager.service
NEEDRESTART-SVC: packagekit.service
